libcxx/include/__config

```diff
@@ -460,6 +460,10 @@ _LIBCPP_HARDENING_MODE_DEBUG
 #    define __has_constexpr_builtin(x) 0
 #  endif
 
+#  ifndef __building_module
```

Comment:
Please add comments to this `#ifdef` the name is very unfortunate, this seems to be about clang modules and not C++20 modules.